sidebars.json

```diff
@@ -0,0 +1,141 @@
+{
+    "docs": [
+        { "type": "link", "label": "Introduction", "href": "/" },
+        "getting-started",
```

Comment:
Could we also somewhere still have a full overview page listing all articles in a list similar to what we have currently? If you want a total overview of all the articles this sidebar is not really easy to navigate.

Reply:
There is no builtin way to do that, but it might be possible by:
1. [Swizzling](https://v2.docusaurus.io/docs/using-themes/#swizzling-theme-components) some page-related component
1. Getting sidebar list from props and putting it to [React Context](https://reactjs.org/docs/context.html)
1. Creating a list renderer component (which would have to effectively reimplement sidebar logic)

That's not too hard, but still some extra complexity. Maybe we could just make sidebar easier to navigate, for example by increasing it's width?

Reply:
I don't think the width is really the problem, moreso the height and collapsing. I think simply implementing a component that just prints out the list from this sidebars.json file with links is fine.